lineprotocol/decoder.go

```diff
@@ -24,7 +24,9 @@ const (
 )
 
 var (
-	fieldSeparatorSpace   = newByteSet(" ")
+	// Note: in some places we hard-code a single space in the source for efficency.
```

Comment:
it would mean the rest of the source would no longer be consistent with this value (hence the comment, which i actually had intended to add in PR #53 ), which is arguably "breaking". Given that the spec is fixed at this point, I don't think anyone should be changing this, but worth leaving this comment in case anyone feels tempted.